### PR TITLE
Add march news

### PR DESCRIPTION
**Pull request by dbu**
@dantleech if the hackday will happen, we should announce that too.

### CODE REVIEW

**Comment by dantleech** [comment]
Yeah we should find out more today. I will also try and add some stuff here.

**Comment by dbu** [comment]
thanks. just wanted to start something. we should also say something about jackalope release (and make that release happen very soon)

**Comment by lsmith77** [comment]
guess we have a date now?

**Comment by dantleech** [comment]
I have added some more news, still need to get some confirmation from @chirimoya about the details for the hackday.

**Comment by dantleech** [comment]
ok, I have some more details although the exact venue has yet to be decided. Have added a separate post  about the hackday.

File: src/Cmf/MainBundle/Resources/data/posts/2015-03-18-may-hackday.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  the 1st and 2nd of May. Friday will be a presentation / workshop day, and
+  Saturday will be a more general day for hacking and discussion. 
+  
+  You can learn more about the present and future of the CMF, Sulu CMS and
+  Jackalope (PHPCR) and get hands on experience.
+
+  The venue and catering will be provided by MassiveArt webservices and there
+  will be a social event on Friday evening.
+
+  There will be stickers and t-shirts.
```
Comment (dantleech) [inline src/Cmf/MainBundle/Resources/data/posts/2015-03-18-may-hackday.yml]: Should add the doodle link and some details about how to get to Dornbirn .. /cc @chirimoya @berndhep

File: src/Cmf/MainBundle/Resources/data/posts/2015-03-18-may-hackday.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: "may-hackday-2015"
+parent: "/news"
+title: "Symfony CMF - Sulu hackday May 2015"
+label: false
+format: markdown
+template: CmfMainBundle:Cms:news_detail.html.twig
+publish_start_date: "2015-03-16"
```
Comment (dbu) [inline src/Cmf/MainBundle/Resources/data/posts/2015-03-18-may-hackday.yml]: maybe give this a bit a later date?

File: src/Cmf/MainBundle/Resources/data/posts/2015-03-16.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
+name: "cmf-news-march-2015"
+parent: "/news"
+title: "CMF News March 2015"
+label: false
+format: markdown
+template: CmfMainBundle:Cms:news_detail.html.twig
+publish_start_date: "2015-03-16"
```
Comment (dbu) [inline src/Cmf/MainBundle/Resources/data/posts/2015-03-16.yml]: we should update this to "today" before we merge.

File: src/Cmf/MainBundle/Resources/data/posts/2015-03-18-may-hackday.yml (hunk 1 above)
Comment (dbu) [inline src/Cmf/MainBundle/Resources/data/posts/2015-03-18-may-hackday.yml]: maybe create a subscription on something like techup.ch where people can click that they come? or you want a doodle to select each day and other options?

**Comment by dbu** [comment]
shall we take the news about the hackday out of this update and publish the news with just a pre-announcement for the hackday? then publish the hackday news when we have RSVP and all ready?

and i just notice we should mention the symfonycmf twitter account that we created.

**Comment by dantleech** [comment]
thats fine with me

**Comment by dbu** [comment]
okay, i will tackle this tomorrow morning

**Comment by dbu** [comment]
hackday moved to #64

**Pull request merged**